### ISSUE TITLE
🐛 Bug Report: Can no longer search through memberships

### ISSUE THREAD

**Issue by JakeAi** [appwrite/appwrite#2789]
### 👟 Reproduction steps

In 0.11, you used to be able to 
```TS
getMemberships(teamId, 'userId='+account.$id)
```

In 0.12, searching is omitted from [teams.php](https://github.com/appwrite/appwrite/blob/0.12.x/app/controllers/api/teams.php#L422)

### 👍 Expected behavior

Return memberships filtered by search

### 👎 Actual Behavior

No search filtering happened

### 🎲 Appwrite version

Version 0.12.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by TorstenDittmann** [comment]
Hey 👋🏻 

As of right now you can just search for the memberships using the `$id`, `email` and `name` 👍🏻 
```
getMemberships(teamId, account.$id)
getMemberships(teamId, account.email)
getMemberships(teamId, account.name)
```

**Comment by JakeAi** [comment]
> 

This returns all memberships for the team. No filtering is done.

**Comment by TorstenDittmann** [comment]
Oh, good catch!

`$search` is actually not used 👍🏻  gonna work on a fix.

**Comment by JakeAi** [comment]
> Oh, good catch!
> 
> `$search` is actually not used 👍🏻 gonna work on a fix.

I see 0.12.3 is released, I pulled the new docker images. How does the search work now?

**Comment by eldadfux** [comment]
I think this would get a fix with the 0.13 data. migration

**Comment by JakeAi** [comment]
> I think this would get a fix with the 0.13 data. migration

Is there a sql command I can run to fix it so I don’t have to update my android and iOS apps?

**Comment by TorstenDittmann** [comment]
We just released 0.13 and this should be fixed 👍🏻 

You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).

Thank you a lot for reporting this 🙏🏻 

@JakeAi sadly this wasn't a Database Issue - but also an API issue which cannot be fixed with only SQL.

**Comment by JakeAi** [comment]
> We just released 0.13 and this should be fixed 👍🏻
> 
> You can find the announcement [here](https://dev.to/appwrite/announcing-appwrite-013-with-major-upgrades-to-storage-and-functions-3hpf).
> 
> Thank you a lot for reporting this 🙏🏻
> 
> @JakeAi sadly this wasn't a Database Issue - but also an API issue which cannot be fixed with only SQL.

I can confirm the issue is fixed. Thanks!
